### DISCUSSION TITLE
Set the working copy to a specific change

### DISCUSSION THREAD

**Discussion by stouset** [jj-vcs/jj#3056] [Q&A] [answered]
Let's say I've run `jj new` to work on a new set of changes (let's call that change`b` and its parent `a`). I switch away from those and go work on something else with `jj new {somewhere else}`. How can I *go back* to continue editing the change I was previously working on (`b`)? If I run `jj new b` I end up on a new set of changes (`c`). If I run `jj new a`, I end up on a parallel set of changes that doesn't include `b`.

How can I set the working directory and the current changeset being edited explicitly to an existing change?

**Comment by yuja** [comment, marked as answer]
`jj edit b` is the command to check out `b` as the working copy commit.

**Comment by stouset** [comment]
That'll do it, thanks.
